packages/core/docs/general/error-handling.md

```diff
@@ -0,0 +1,85 @@
+# Error handling
+
+A flexible way of error handling is essential for a framework like Vue Storefront. As factories of composables are hearth of our core - we decided to put there whole error handling mechanism.
```

Comment:
Factories are internal functionality. Most of the users reading the docs that never got that deep will not be aware of what they are and could get confused